MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs

```diff
@@ -3,6 +3,6 @@ namespace MvvmCross.iOS.Views.Presenters.Attributes
 {
     public class MvxRootPresentationAttribute : MvxBasePresentationAttribute
     {
-        public bool WrapInNavigationController { get; set; }
+		public bool WrapInNavigationController { get; set; } = true;
```

Comment:
I see that would not be nice indeed. I have added a new commit correcting this problem.